src/components/advanced-color-picker/advanced-color-picker-cell.style.js

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components';
+
+const transparentSvg = '%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%'
```

Comment:
`transparentSvg` is declared 4 times in different files with the same value, maybe save it as a svg file somewhere?

Reply:
thanks, created svg file.